javascript/javascript3/week1/script.js

```diff
@@ -0,0 +1,143 @@
+let weather = {
+    "apiKey": "fd548f55ee5a12f2b7b63bfc72cd0260",
+    fetchWeather: function (city) {
+        fetch(
+                "https://api.openweathermap.org/data/2.5/weather?q=" +
+                city +
```

Comment:
Use backtick string formatting (Template literals) instead of string concatenation, then you dont have to worry about escaping characters